BootloaderCommonPkg/Library/NvmExpressLib/NvmExpress.c

```diff
@@ -376,6 +382,8 @@ NvmeInitialize (
     }
   }
 
+  MmioAnd16 (NvmeHcPciBase + PCI_COMMAND_OFFSET, (UINT16)~EFI_PCI_COMMAND_BUS_MASTER );

```

Comment:
I think you can handle the whole command register  I/O+MEM+BusMaster together.

Reply:
Thanks and resolved